### DISCUSSION TITLE
Resizing images

### DISCUSSION THREAD

**Discussion by watson-e-and** [gavvvr/obsidian-imgur-plugin#58] [Getting help]
Is there a clean way to resize the images once they've been pasted into my Vault? 

I love this plugin, and it feels like this is a feature that might be there, but I just don't know how.

**Comment by gavvvr** [comment]
Hi @watson-e-and 
This should be a job for some general plugin working with any images (not necessarily tied to Imgur)
This one ([mousewheel-image-zoom](https://github.com/nicojeske/mousewheel-image-zoom)) is what you need. I've just tested it and it works fine with remote images uploaded by Imgur plugin too.

Also, there is a functionality built into Imgur plugin to use previews which Imgur generates.
Just place your cursor onto Imgur image Markdown code and press `Cmd/Ctrl + P` to open "Command palette" and type "resize". You will see options to get Imgur-generated thumbnails and previews:
<img width="499" alt="image" src="https://github.com/gavvvr/obsidian-imgur-plugin/assets/1719646/c6e27625-3d40-4fb3-a86d-c4ae5adb560b">
